draft-ietf-quic-http.md

```diff
@@ -983,7 +983,7 @@ Push ID:
   ({{frame-cancel-push}}), and PRIORITY frames ({{frame-priority}}).
 
 Header Block:
-: QPACK-compressed request headers for the promised response.  See [QPACK] for
+: QPACK-compressed request header for the promised response.  See [QPACK] for
```

Comment:
I think that we can say header fields here.  QPACK doesn't compress the header, it only compresses a list of header fields.

Reply:
I'd call semantics on the distinction, since the list of header fields is the header, but I think it reads better your way.